.github/workflows/prepare-for-prod-deploy.yml

```diff
@@ -101,20 +101,31 @@ jobs:
           echo "Transformer: $TAG_NAME"
           echo "User Transformer: $UT_TAG_NAME"
 
+      - name: Clone Devops Repo
+        run: |
+          git clone https://${{secrets.PAT}}@github.com/rudderlabs/rudder-devops.git
+
+      - name: Extract branch name
+        id: extract_branch_name
+        run: |
+          cd rudder-devops
+          branch_name=$(git rev-parse --abbrev-ref HEAD)
+          echo "branch_name=$branch_name"
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+
       - name: Update Helm Charts and Raise Pull Request
         env:
           GITHUB_TOKEN: ${{ secrets.PAT }}
         run: |
-          git clone https://${{secrets.PAT}}@github.com/rudderlabs/rudder-devops.git
           cd rudder-devops
-          git checkout -b shared-transformer-$TAG_NAME
+          git checkout -b shared-transformer-$TAG_NAME ${{steps.extract_branch_name.outputs.branch_name}}
```

Comment:
`shared-transformer-$TAG_NAME` itself is the branch name, why computing `branch_name` again at L112 ?
And there's space b/w two parameters, this command will fail

Reply:
This will create shared-transformer-$TAG_NAME branch from **master** branch of devops repo.
`git checkout -b feature-branch main`

`${{steps.extract_branch_name.outputs.branch_name}}` is not required in Shared transformer step.  As by default it will take the base branch from clone repo.
But this will be required in creating hosted transformer branch from master branch.

Removing `${{steps.extract_branch_name.outputs.branch_name}}` from shared transformer step